### PR TITLE
hub: do not export unwanted XML-RPC methods

### PR DESCRIPTION
**Pull request by lzaoral**
Resolves: #109

### CODE REVIEW

File: osh/hub/osh_xmlrpc/worker.py [hunk 1]
```diff
@@ -27,6 +27,28 @@
...
+    'fail_task',
+    'finish_analyzers_version_retrieval',
+    'finish_scan',
+    'finish_task',
+    'get_scanning_args',
+    'get_su_user',
+    'interrupt_tasks',
+    'move_upload',
+    'set_scan_to_basescanning',
+    'set_scan_to_scanning',
```
Comment (rhyw) [inline osh/hub/osh_xmlrpc/worker.py:49]: I confirmed that some of the methods are indeed gone with this patch, e.g., the `worker.AnalyzerVersion`. However I still see some of inconsistencies, i.e., I see around 34 "worker.xxx" methods under `/xmlrpc/worker/` while here I see only 17.

For example I see `worker.create_subtask`/`worker.assign_task` etc.

Comment (lzaoral) [reply osh/hub/osh_xmlrpc/worker.py:49]: The default set of XML-RPC methods is exported by `kobo`: https://github.com/release-engineering/kobo/blob/master/kobo/hub/xmlrpc

OSH either adds new methods or overrides the ones from `kobo` to add some extra functionality.  The set of all exported Python modules is defined here: https://github.com/openscanhub/openscanhub/blob/ac9459c863ecc62bcd135f7db18b56a311c0c020/osh/hub/settings.py#L152-L178

Comment (rhyw) [reply osh/hub/osh_xmlrpc/worker.py:49]: aha I see, thanks

**Review by rhyw — APPROVED** [review]

**Comment by lzaoral** [comment]
Thank you for the review, @rhyw!

**Pull request merged**